doc/release-notes.md

```diff
@@ -87,6 +106,13 @@ New settings
 Wallet
 ------
 
+- To improve wallet privacy, the frequency of wallet rebroadcast attempts has
+  been reduced from approximately once every 15 minutes to once every 12-36
```

Comment:
Each of these three release note entries employs a different verb tense: future ("will try/will not track") in the first entry, present ("now returns/tracks/returns") in the second one, and a form of past ("has been reduced") tense at the beginning of the third one. Perhaps use a consistent verb tense.

Reply:
hm, I updated this one to be present ("has been reduced" -> "is reduced"). 

I read through the other ones and the difference between present & future makes sense to me... eg.
> The mempool now tracks whether transactions submitted via the wallet or RPCs
  have been successfully broadcast. Every 10-15 minutes, the node will try to
  announce unbroadcast transactions until...

the present "now tracks" indicates how the code has changed, but the future "will try" is what happens when you run the code in the future. 

so I've left them for now. if you have any specific suggestions, I'm happy to consider them.